std-bits/base/src/main/java/org/enso/base/ObjectComparator.java

```diff
@@ -1,73 +1,69 @@
 package org.enso.base;
 
+import org.graalvm.polyglot.Context;
 import org.graalvm.polyglot.Value;
 
 import java.time.LocalDate;
 import java.time.LocalTime;
 import java.time.ZonedDateTime;
 import java.util.Comparator;
 import java.util.Locale;
-import java.util.function.BiFunction;
 import java.util.function.Function;
+import java.util.function.BiFunction;
 
 public class ObjectComparator implements Comparator<Object> {
-  private static ObjectComparator INSTANCE;
-
-  /**
-   * A singleton instance of an ObjectComparator.
-   *
-   * @param fallbackComparator this MUST be the default .compare_to function for Enso. Needs to be
-   *     passed to allow calling back from Java.
-   * @return Comparator object.
-   */
-  public static ObjectComparator getInstance(Function<Object, Function<Object, Value>> fallbackComparator) {
-    if (INSTANCE == null) {
-      INSTANCE = new ObjectComparator(fallbackComparator);
+  public static final ObjectComparator DEFAULT = new ObjectComparator();
+
+  private static BiFunction<Object, Object, Integer> EnsoCompareCallback = null;
```

Comment:
Isn't this field name against naming conventions? Names are supposed to start with lower case.

Reply:
+1